application/blueprint/services/continuous_planning_service.py

```
@@ -729,35 +862,32 @@ def _parse_llm_response(self, response) -> Dict:
+        cleaned = _sanitize_llm_json_output(content)
+        cleaned = _extract_outer_json_value(cleaned)
+        try:
+            return json.loads(cleaned)
+        except json.JSONDecodeError:
+            pass
+
+        try:
+            repaired = repair_json(cleaned)
+            return json.loads(repaired)
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)